packages/api/apollo-server/types/index.ts

```diff
@@ -0,0 +1,63 @@
+import { gql } from 'apollo-server-core'
+
+const Types = gql`
+  type Query {
+    users: [User]
+  }
+
+  type Mutation {
+    userMutation: [User]
+  }
+
+  enum UserType {
+    TA
+    STUDENT
+    INSTRUCTOR
+  }
+
+  type User {
+    id: ID!
+    firstName: String
+    lastName: String
+    utorID: String!
+    email: String
+    userType: UserType!
+    lastLogin: String
+    isActive: Boolean
```

Comment:
why is `lastLogin` and `isActive` optional?

Reply:
> why is `lastLogin` and `isActive` optional?

`isActive` should be required, but the `lastLogin` should be `null` before the first time users login.

Reply:
IMO, it should be initialized to the created date when the user is created.